cmd/root.go

```diff
@@ -609,6 +610,13 @@ func newRunCommand(ctx context.Context, input *Input) func(*cobra.Command, []str
 
 		cancel := artifacts.Serve(ctx, input.artifactServerPath, input.artifactServerAddr, input.artifactServerPort)
 
+		// TODO: provide a way to configure the cache handler via cli
+		cacheHandler, err := artifactcache.StartHandler("", "", 0, common.Logger(ctx))
```

Comment:
Using '' as the default `dir` value will implicitly store probably gigabytes of data in the users homedir without users noticing.
Should this be better a custom temp folder which is cleanup by the OS?

Don't get me wrong, it should be configurable, but by default we should be more conservative with users diskspace.

Reply:
'' means default path (`$HOME/.cache/actcache`) for `artifactcache. StartHandler`, and I made it configurable now:

<img width="1356" alt="image" src="https://user-images.githubusercontent.com/9418365/234832828-19d07e9f-c060-4db6-b3d5-782a45f8c9ea.png">

So default value could be `$XDG_CACHE_HOME/actcache` or `$HOME/.cache/actcache`, I think it's fair enought, right?

And don't worry, artifactcache will clean the old cache even the OS does't do it.